Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs

```diff
@@ -309,6 +332,21 @@ public static void NodeRaysTransform(ref IntPtr node, Matrix4x4 transform)
             }
         }
 
+        public static void NodeRaysVelocityDistortion(ref IntPtr node, Vector3 velocity, Vector3 angularVelocity)
```

Comment:
```suggestion
        public static void NodeRaysVelocityDistortion(ref IntPtr node, Vector3 linearVelocity, Vector3 angularVelocity)
```
And please propagate it over the plugin.